pkg/cmd/prune.go

```diff
@@ -0,0 +1,85 @@
+// Copyright 2024 Daytona Platforms Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+package cmd
+
+import (
+	"github.com/daytonaio/daytona/cmd/daytona/config"
+	workspace "github.com/daytonaio/daytona/pkg/cmd/workspace"
+	view "github.com/daytonaio/daytona/pkg/views/prune"
+	view_util "github.com/daytonaio/daytona/pkg/views/util"
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+)
+
+var forceFlag bool
```

Comment:
I'd change this to `-y` (yesFlag) because the prompt itself is a yes/no. Then there's no need for the `confirmCheck` var.

Reply:
Okay, agreed.